app/lib/activitypub/tag_manager.rb

```diff
@@ -13,7 +13,9 @@ class ActivityPub::TagManager
   }.freeze
 
   def public_collection?(uri)
-    uri == COLLECTIONS[:public] || uri == 'as:Public' || uri == 'Public'
+    %w(as:Public Public)
+      .push(COLLECTIONS[:public])
+      .include?(uri)
```

Comment:
Has this been auto-corrected by rubocop? If yes, I find this baffling.

This is a *lot* less readable than the original and needs to allocate an array where previously none was needed. If this is a new cop, I would say let us please disable it.

Reply:
I had left this as a note before but may have rebased myself out of it.

The rubocop autocorrect produced:

```
  def public_collection?(uri)
    ['as:Public', 'Public'].include?(uri)
  end
```

Note how that drops the `COLLECTIONS[:public]` from the original.

This is the cop in question - https://docs.rubocop.org/rubocop/cops_style.html#stylemultiplecomparison

The version here was my attempt to fix the autocorrect while still satisfying the cop.

That said - I think I like disable better. Pushed to change to that.

Reply:
I sure could. https://github.com/rubocop/rubocop/issues/13553